Grbl_Esp32/src/GCode.cpp

```diff
@@ -197,113 +193,165 @@ uint8_t gc_execute_line(char* line, uint8_t client) {
+                                mg_word_bit = ModalGroup::MG4;
+                                break;
+                            default: FAIL(STATUS_GCODE_UNSUPPORTED_COMMAND); break;
                         }
                         break;
                     case 93:
+                        gc_block.modal.feed_rate = FeedRate::InverseTime;
+                        mg_word_bit              = ModalGroup::MG5;
+                        break;
                     case 94:
-                        word_bit                 = MODAL_GROUP_G5;
-                        gc_block.modal.feed_rate = 94 - int_value;
+                        gc_block.modal.feed_rate = FeedRate::UnitsPerMin;
+                        mg_word_bit              = ModalGroup::MG5;
                         break;
                     case 20:
+                        gc_block.modal.units = Units::Inches;
+                        mg_word_bit          = ModalGroup::MG6;
+                        break;
                     case 21:
-                        word_bit             = MODAL_GROUP_G6;
-                        gc_block.modal.units = 21 - int_value;
+                        gc_block.modal.units = Units::Mm;
+                        mg_word_bit          = ModalGroup::MG6;
                         break;
                     case 40:
-                        word_bit = MODAL_GROUP_G7;
                         // NOTE: Not required since cutter radius compensation is always disabled. Only here
                         // to support G40 commands that often appear in g-code program headers to setup defaults.
-                        // gc_block.modal.cutter_comp = CUTTER_COMP_DISABLE; // G40
+                        // gc_block.modal.cutter_comp = CutterCompensation::Disable; // G40
+                        mg_word_bit = ModalGroup::MG7;
                         break;
                     case 43:
                     case 49:
-                        word_bit = MODAL_GROUP_G8;
                         // NOTE: The NIST g-code standard vaguely states that when a tool length offset is changed,
                         // there cannot be any axis motion or coordinate offsets updated. Meaning G43, G43.1, and G49
                         // all are explicit axis commands, regardless if they require axis words or not.
-                        if (axis_command)
+                        if (axis_command != AxisCommand::None)
                             FAIL(STATUS_GCODE_AXIS_COMMAND_CONFLICT);
                         // [Axis word/command conflict] }
-                        axis_command = AXIS_COMMAND_TOOL_LENGTH_OFFSET;
+                        axis_command = AxisCommand::ToolLengthOffset;
                         if (int_value == 49)  // G49
-                            gc_block.modal.tool_length = TOOL_LENGTH_OFFSET_CANCEL;
+                            gc_block.modal.tool_length = ToolLengthOffset::Cancel;
                         else if (mantissa == 10)  // G43.1
```

Comment:
{...} braces

Reply:
I just fixed all the braces in that file.  Dozens of them.